core/components/css-utilities/Display/Display.story.tsx

```diff
@@ -75,28 +69,289 @@ export const display = () => {
+            <Text appearance="default" className="p-3 bg-secondary-lightest border">
+              Flex item 2
+            </Text>
+            <Text appearance="default" className="p-3 bg-secondary-lightest border">
+              Flex item 3
+            </Text>
+          </div>
+        </div>
+        <Text appearance="subtle" className="mb-2 font-size-xs">
+          Elements with d-flex become flex containers, with their children arranged in a row by default.
+        </Text>
+        <div className="bg-secondary-lightest p-3 rounded-10 font-size-xs">
+          <pre className="m-0">
+            <code className="d-block">{'<div className="d-flex bg-secondary-lightest border p-4 gap-3">'}</code>
+            <code className="d-block mt-2">
+              {'  <Text appearance="default" className="p-3 bg-secondary-lightest border">Flex item 1</Text>'}
+            </code>
+            <code className="d-block mt-2">
+              {'  <Text appearance="default" className="p-3 bg-secondary-lightest border">Flex item 2</Text>'}
+            </code>
+            <code className="d-block mt-2">
+              {'  <Text appearance="default" className="p-3 bg-secondary-lightest border">Flex item 3</Text>'}
+            </code>
+            <code className="d-block mt-2">{'</div>'}</code>
+          </pre>
+        </div>
+      </div>
+
+      {/* d-inline-block example */}
+      <div className="mt-8 mb-8">
+        <Heading size="s" className="mb-3">
+          Inline Block Display
+        </Heading>
+        <div className="mb-4">
+          <div className="d-inline-block bg-secondary-lightest border p-3 mr-2 mb-2 w-50 h-25 align-top">
+            d-inline-block
+            <br />
+            w-50 class
+            <br />
+            h-25 class
+          </div>
+          <div className="d-inline-block bg-secondary-lightest border p-3 mr-2 mb-2 w-25 h-50 align-top">
+            d-inline-block
+            <br />
+            w-25 class
+            <br />
+            h-50 class
+          </div>
+        </div>
+        <Text appearance="subtle" className="mb-2 font-size-xs">
+          Elements with d-inline-block flow like inline elements but can have width and height like block elements.
+        </Text>
+        <div className="bg-secondary-lightest p-3 rounded-10 font-size-xs">
+          <pre className="m-0">
+            <code className="d-block">
+              {'<div className="d-inline-block bg-secondary-lightest border p-3 mr-2 mb-2 w-25 h-50 align-top">'}
+            </code>
+            <code className="d-block mt-2">{'  d-inline-block'}</code>
+            <code className="d-block mt-2">{'  <br />'}</code>
+            <code className="d-block mt-2">{'  w-25 class'}</code>
+            <code className="d-block mt-2">{'  <br />'}</code>
+            <code className="d-block mt-2">{'  h-50 class'}</code>
+            <code className="d-block mt-2">{'</div>'}</code>
+          </pre>
+        </div>
+      </div>
```

Comment:
Write full code of the example to avoid confusion. Also Instead of using multiple <code> elements we can simply write all the text inside single <code>